### DISCUSSION TITLE
Why labels on my map suddenly got huge?

### DISCUSSION THREAD

**Discussion by matkoniecz** [organicmaps/organicmaps#8274] [Q&A] [answered]
Street and place labels on my map are suddenly much bigger. Why it could have happened?

I am pretty sure that it has not happened directly after update.

Have I enabled somehow some weird setting? How can I get normal size back?

**Comment by hb0nd** [comment, marked as answer]
there's a setting called "increase size for map labels" in settings, maybe you accidentally enabled it?

**Reply by matkoniecz** [reply]
yes, this one! Maybe I put unlocked phone to pocket...
